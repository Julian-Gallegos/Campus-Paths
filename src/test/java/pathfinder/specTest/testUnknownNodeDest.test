# Find a path with Dijkstra's algorithm when
# the destination node does not exist

CreateGraph simpleGraph

AddNode simpleGraph A

FindPath simpleGraph A B