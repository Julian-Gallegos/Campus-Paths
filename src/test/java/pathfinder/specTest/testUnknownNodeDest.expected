# Find a path with Dijkstra's algorithm when
# the destination node does not exist

created graph simpleGraph

added node A to simpleGraph

path from A to B:
unknown node B